pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md

```diff
@@ -14,7 +14,7 @@ Before creating a Kubernetes cluster you must have the required images in a loca
 1.  Download the images bundle.
 
     ```bash
-    curl -o konvoy-image-bundle.tar.gz -O http://downloads.d2iq.com/konvoy/airgapped/v2.1.1/konvoy_image_bundle_v2.1.1_linux_amd64.tar.gz
+    curl --output konvoy-image-bundle.tar.gz --location https://downloads.d2iq.com/konvoy/airgapped/v2.1.1/konvoy_image_bundle_v2.1.1_linux_amd64.tar.gz
```

Comment:
hmm, so we just updated these commands. I'd like to hear from @dkoshkin specifically before we update these commands again.

Reply:
Totes, currently many of them do not work and just download a 0 byte file since they are being 302 redirected by CloudFront and curl does not follow redirects by default. For a while I couldn't figure out why the untar was failing :facepalm:

Reply:
These changes look correct.